api/oas_templates/python/pyproject.mustache

```diff
@@ -43,8 +43,8 @@ algokit-utils = "4.0.0"
 algokit-transact = { path = "../../../packages/python/algokit_transact" }
 
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0"]
```

Comment:
tried a fresh poetry init on latest v2, the version specifier it drops under default build system is:

```
requires = ["poetry-core>=2.0.0,<3.0.0"]
build-backend = "poetry.core.masonry.api"
```

Wonder if we should set the same specifier, considering poetry v1 is fairly old at this point